### PR TITLE
LPS-193397 | Master

### PR DESCRIPTION
**Pull request by kristinonias**
https://liferay.atlassian.net/browse/LPS-193397
https://liferay.atlassian.net/browse/LRQA-74799
https://liferay.atlassian.net/browse/LRQA-74797

cc: @patriciajeanperez

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase [hunk 1]
```diff
@@ -38,24 +38,6 @@ definition {
 
 	}
 
-	@description = "Verify that any user with administrator role can reassign allowed tasks"
```
Comment (kristinonias) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:41]: Removed as it is already covered in another test https://liferay.atlassian.net/browse/LRQA-74797

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase [hunk 2]
```diff
@@ -38,24 +38,6 @@ definition {
...
-	@ignore = "Test Stub"
-	@priority = 5
-	test AdministratorUserCanReassignAnyTasks {
-
-		// TODO AdministratorUserCanReassignAnyTasks pending implementation
-
-	}
-
-	@description = "Verify that an administrator user that it's not asset creator can reassign multiple already assigned tasks to different users"
```
Comment (kristinonias) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:50]: Removed as it is already covered in another test https://liferay.atlassian.net/browse/LRQA-74799

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Comment by patriciajeanperez** [comment]
ci:forward:force

**Pull request closed without merging**